.pipelines/rp-int-deploy.yml

```diff
@@ -0,0 +1,88 @@
+trigger: none
+pr: none
+
+parameters:
+  - template: rp-params.yml
+
+resources:
+  containers:
+    - container: container
+      image: registry.access.redhat.com/ubi8/toolbox:8.8
+      options: --user=0 --privileged -v /dev/shm:/dev/shm --device /dev/net/tun --name vpn
+
+variables:
+  # - template: vars.yml # probably redundant
+  - template: rp-vars.yml
+
+jobs:
+  - job: RP_int_deployment
+    timeoutInMinutes: 180
+    pool:
+      name: 1es-aro-ci-pool
+
+    steps:
+# setup
+      - template: ./templates/template-checkout.yml
+      - template: ./templates/template-az-cli-login.yml # does it have az CLI alraedy?
+        parameters:
+          azureDevOpsJSONSPN: $(aro-v4-e2e-devops-spn)
+      - template: ./templates/rp-int-deployment/template-download-secrets.yml
+      - template: ./templates/rp-int-deployment/template-generate-rp-dev-config.yml
+        parameters:
+          location: ${{ parameters.location }}
+          resourceGroup: ${{ parameters.resourceGroup }}
+          databaseAccountName: ${{ parameters.databaseAccountName }}
+          keyvaultPrefix: ${{ parameters.keyvaultPrefix }}
+          aroImage: ${{ parameters.aroImage }}
+          fluentbitImage: ${{ parameters.fluentbitImage }}
+      - script: make deploy # use different target which will succeed and won't fail
```

Comment:
We should have a predeploy pipeline / "bootstrap" pipeline around somewhere in ev2/ARO-Pipelines that predeploys resources such that make deploy doesn't fail. It's for production use and I'm not sure exactly where it lives, but it might be worth tracking down for the possibility of re-use (or at least figuring out what resources need to be pre-created and how) if you don't want to go down the path of retrying / looping `make deploy` knowing that it will fail in the middle.

Reply:
Sounds interesting